app/controllers/ingestibles_controller.rb

```diff
@@ -71,25 +71,18 @@ def rmauth
 
   # PATCH/PUT /ingestibles/1 or /ingestibles/1.json
   def update
-    respond_to do |format|
-      if @ingestible.update(ingestible_params)
-        format.html { redirect_to edit_ingestible_url(@ingestible), notice: 'Ingestible was successfully updated.' }
-        format.json { render :edit, status: :ok, location: @ingestible }
-      else
-        format.html { render :edit, status: :unprocessable_entity }
-        format.json { render json: @ingestible.errors, status: :unprocessable_entity }
-      end
+    if @ingestible.update(ingestible_params)
+      flash.now.notice = t('.success')
+      render :edit
```

Comment:
used `render :edit` instead of `redirect to edit`, should be a little bit faster